docs/UserGuide.md

```diff
@@ -542,7 +517,12 @@ Our application is **case sensitive**. This means that item names must match exa
 
 Ensure you use the correct capitalization when interacting with items in the inventory.
 
----
+
+### No special characters
+
+Our application **does not support special characters** for any input.
+
+Preferably, please keep to **lowercase, alphanumeric characters** for command inputs.
```

Comment:
We should doublecheck that all our sample inputs are also lowercase, just to match with this